0-getting-started/install/centos.md

```diff
@@ -19,7 +21,56 @@ sudo wget http://download.rethinkdb.com/centos/6/`uname -m`/rethinkdb.repo \
 sudo yum install rethinkdb
 ```
 
-# Compile from source #
+## For Centos 7
+
+```bash
+sudo wget http://download.rethinkdb.com/centos/7/`uname -m`/rethinkdb.repo \
```

Comment:
The links for Centos 6 and Centos 7 should use `https` I think. The Centos 6 change for this is in https://github.com/rethinkdb/docs/pull/871 , but it hasn't been merged yet.